src/materials/MeshPhongMaterial.js

```diff
@@ -94,6 +97,9 @@ function MeshPhongMaterial( parameters ) {
 	this.reflectivity = 1;
 	this.refractionRatio = 0.98;
 
+	this.toonMap = null;
```

Comment:
Reference: http://gamedev.stackexchange.com/questions/51063/what-is-ramp-shading-or-lighting

`rampMap` would be better than `toonMap`?